rules/windows/defense_evasion_sdelete_like_filename_rename.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects file name patterns as caused by the use of Sysinternals Sdelete utility to securely delete a file via multiple
+file overwrite and rename operations. Verify process details such as command line and hash to confirm this activity
+legitimacy.
```

Comment:
I think the notes field would be a more suitable place for this sentence.

> Verify process details such as command line and hash to confirm this activity legitimacy.

Reply:
done